managing_providers/_topics/embedded_workflows.md

```diff
@@ -181,6 +182,9 @@ Workflows must be authored in Amazon State Languages (ASL) format. As part of au
 
    You are recommended to use a docker.io [access token](https://docs.docker.com/security/for-developers/access-tokens/) so that the token does not expire.
 
+   b) Provide an image pull secret to a podified Kubernetes container, and then add it to a service account
+     In order to pull an image from a private registry you have to provide an `ImagePullSecret` to your containers, see [Pull an Image from a Private Registry](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).  {{ site.data.product.title_short }} uses a service account called `manageiq-default` to run containers for your workflows.  You can add an `ImagePullSecret` to this service account by following [Add Image Pull Secrets to a service account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account/).
```

Comment:
Same comment here for the `manageiq-default` - if you need it productized, it should use a `{{}}` token here.

Reply:
Also isn't productized as far as I know cc @bdunne

Reply:
Oh I misread the OP then

Reply:
It turns out _this_ one is productized but not the podman one.

Reply:
manageiq-default is productized

Reply:
We're working on the conref/equivalent format used for ManageIQ for this. Will update once done!